app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] When a user clicks the "forgot your password?" link there is an error, also I encourage you to work on all the app links, the app should work without any link breaking.

![pass](https://user-images.githubusercontent.com/53124548/206209926-a3ca6a3b-dedf-453c-b25b-ff3d8d514c22.png)